src/components/sbb-alert/sbb-alert.tsx

```diff
@@ -0,0 +1,238 @@
+   */
+  @Prop({ reflect: true }) public readonly = false;
+
+  /** You can choose between `m` or `l` size. */
+  @Prop({ reflect: true }) public size: InterfaceAlertAttributes['size'] = 'm';
+
+  /** Whether the fade in animation should be disabled. */
+  @Prop() public disableAnimation = false;
+
+  /**
+   * Name of the icon which will be forward to the nested `sbb-icon`.
+   * Choose the icons from https://lyne.sbb.ch/tokens/icons/.
+   * Styling is optimized for icons of type HIM-CUS.
+   */
+  @Prop() public iconName = 'info';
+
+  /** Content of title. */
+  @Prop() public titleContent?: string;
+
+  /** Level of title, will be rendered as heading tag (e.g. h3). Defaults to level 3. */
+  @Prop() public titleLevel: InterfaceTitleAttributes['level'] = '3';
+
+  /** This will be forwarded as aria-label to the relevant nested element. */
+  @Prop() public accessibilityLabel: string | undefined;
+
+  /** This will be forwarded as aria-describedby to the relevant nested element. */
+  @Prop() public accessibilityDescribedby: string | undefined;
+
+  /** This will be forwarded as aria-labelledby to the relevant nested element. */
+  @Prop() public accessibilityLabelledby: string | undefined;
+
+  /** The href value you want to link to. */
+  @Prop() public href: string | undefined;
+
+  /** The relationship of the linked URL as space-separated link types. */
+  @Prop() public rel: string | undefined;
+
+  /** Where to display the linked URL. */
+  @Prop() public target: LinkTargetType | string | undefined;
+
+  /** Content of the link */
+  @Prop() public linkContent?: string;
+
+  /** Emits when the fade in animation starts. */
+  @Event({
+    eventName: 'sbb-alert_will-present',
+  })
+  public willPresent: EventEmitter<void>;
+
+  /** Emits when the fade in animation ends and the button is displayed. */
+  @Event({
+    eventName: 'sbb-alert_did-present',
+  })
+  public didPresent: EventEmitter<void>;
+
+  /** Emits when dismissal of an alert was requested. */
+  @Event({
+    eventName: 'sbb-alert_dismissal-requested',
+  })
+  public dismissalRequested: EventEmitter<void>;
+
+  private _transitionWrapperElement!: HTMLElement;
+  private _alertElement!: HTMLElement;
+
+  private _currentLangauge = getDocumentLang();
+  private _firstRenderingDone = false;
+
+  public componentDidRender(): void {
+    if (!this._firstRenderingDone) {
+      this._present();
+    }
+    this._firstRenderingDone = true;
+  }
+
+  /** Requests dismissal of the alert. */
+  @Method() public async requestDismissal(): Promise<void> {
+    this.dismissalRequested.emit();
+  }
+
+  /** Present the alert. */
+  private _present(): Promise<void> {
+    this.willPresent.emit();
+
+    if (this.disableAnimation) {
+      this._onHeightTransitionEnd();
+      return;
+    }
+
+    this._transitionWrapperElement.style.height = `${this._alertElement.offsetHeight}px`;
+    this._transitionWrapperElement.addEventListener(
```

Comment:
Should this maybe be before the assignment to avoid timing issues?

Reply:
here I don't understand you ;)

Reply:
Now I understand :) changed it..